mmv1/products/accesscontextmanager/ServicePerimeter.yaml

```diff
@@ -649,7 +648,6 @@ properties:
                   name: 'sourceRestriction'
                   description: 'Whether to enforce traffic restrictions based on `sources` field. If the `sources` field is non-empty, then this field must be set to `SOURCE_RESTRICTION_ENABLED`.'
                   values:
-                    - :SOURCE_RESTRICTION_UNSPECIFIED
```

Comment:
I think we need to keep these for backwards compatibility, just in case someone managed to put this in a config we can't remove it out from under them

Reply:
Fixed